rest/models/src/main/java/com/connexta/store/rest/models/AddMetadataRequest.java

```diff
@@ -0,0 +1,82 @@
+package com.connexta.store.rest.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class AddMetadataRequest {
```

Comment:
✏️ Now that we are not using the openapi generation anymore, I think we should clean up these models to use constructor arguments or a builder, instead of an empty constructor, set, add, and metadata methods) to validate that the required pieces are present.

Reply:
I would like to see us switch to Lombok for the boiler plate stuff like equals()/hash() and constructors. I don't think it needs to be part of this PR. I'm willing to wait on the cleanup until later.